integration-test/1062-road-shield-cleanup.py

```diff
@@ -15,14 +15,28 @@ def _check_network_relation(
             {'id': way_id, 'shield_text': expected_shield_text})
 
     def test_A151(self):
-        self._check_network_relation(
-            way_id=208288552, rel_id=1159812, tile=(16, 32949, 22362),
-            expected_shield_text='A151')
+        tile = (16, 32949, 22362)
+        self.load_fixtures([
+            'https://www.openstreetmap.org/way/%d' % (208288552,),
+            'https://www.openstreetmap.org/relation/%d' % (1159812,),
+        ], clip=self.tile_bbox(*tile))
+
+        z, x, y = tile
+        self.assert_has_feature(
+            z, x, y, 'roads',
+            {'id': type(None), 'shield_text': 'A151'})
```

Comment:
@nvkelso basically I copied the implementation of `_check_network_relation` and the only diffrence I made here is to assert `'id': type(None)` instead.

Reply:
LGTM